woocommerce/woocommerce
# Add missing dependencies & include @wordpress/components style for storybook

### All Submissions:






### Changes proposed in this Pull Request:



- Add `copy-webpack-plugin` and `url-loader` to root package.json for storybook webpack. They're required to build storybook.
- Include @wordpress/components style for storybook to display component UI properly.

before include @wordpress/components style:
![Screen Shot 2022-05-27 at 10 22 41](https://user-images.githubusercontent.com/4344253/170616911-8ee784bb-0366-4360-84fc-4fd8cafa1ad1.png)

after:
![Screen Shot 2022-05-27 at 10 21 59](https://user-images.githubusercontent.com/4344253/170616921-9fd35324-4947-46a4-ae5e-e0cb7013d31f.png)

### How to test the changes in this Pull Request:

1. Remove storybook cache if needed via `rm -r node_modules/.cache/storybook`
2. Run `pnpm install`
3. Run `storybook`
4. Observe that @wordpress/components style is loaded.

### Other information:




### FOR PR REVIEWER ONLY:
